relay-event-schema/src/protocol/span.rs

```diff
@@ -267,6 +267,10 @@ pub struct SpanData {
     #[metastructure(field = "ai.responses")]
     pub ai_responses: Annotated<Value>,
 
+    /// The integration which emitted a span
+    #[metastructure(field = "sdk.integration", pii = "false")]
+    pub sdk_integration: Annotated<Value>,
```

Comment:
Is this field only used for AI? Seems like a very generic field.

Reply:
So far, but it can be added to other SDKs in the future if we want to filter spans based on which integration they came from